pkg/resource/claim_reconciler.go

```diff
@@ -331,9 +338,26 @@ func (r *ClaimReconciler) Reconcile(req reconcile.Request) (reconcile.Result, er
 	}
 
 	if !meta.WasCreated(managed) {
+		portable := r.newPortableClass()
+		// Portable class reference should always be set by the time we get this far;
+		// Our watch predicates require it.
+		p := types.NamespacedName{
+			Namespace: claim.GetNamespace(),
+			Name:      claim.GetPortableClassReference().Name,
+		}
+		if err := r.client.Get(ctx, p, portable); err != nil {
+			// If we didn't hit this error last time we'll be requeued
+			// implicitly due to the status update. Otherwise we want to retry
+			// after a brief wait, in case this was a transient error or the
+			// portable class is (re)created.
+			claim.SetClassReference(portable.GetClassReference())
```

Comment:
I thought about this, but realized that it may be useful to be able to see the strongly-typed class when `kubectl describe` the claim. This is similar to the pattern with setting the managed resource reference on the claim... which reminds me that I need to update `ResourceClaimSpec` to reference a portable class. Do you feel strongly that the strongly-typed class should be completely removed from the spec?